src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/SortSamSparkIntegrationTest.java

```diff
@@ -38,21 +40,17 @@ public void testSortBAMs(
         final File actualOutputFile = createTempFile("sort_sam", outputExtension);
         File referenceFile = null == referenceFileName ? null : new File(getTestDataDir(), referenceFileName);
         ArgumentsBuilder args = new ArgumentsBuilder();
-        args.add("--input"); args.add(inputFile.getCanonicalPath());
-        args.add("--output"); args.add(actualOutputFile.getCanonicalPath());
+        args.addInput(inputFile);
+        args.addOutput(actualOutputFile);
         if (null != referenceFile) {
-            args.add("--R");
-            args.add(referenceFile.getAbsolutePath());
+            args.addReference(referenceFile);
         }
-        args.add("--num-reducers"); args.add("1");
+        args.addArgument(GATKSparkTool.NUM_REDUCERS_LONG_NAME, "1");
```

Comment:
Why does num reducers need to be 1 for this test?

Reply:
I don't know...  Any number should be fine I think.